src/js/client/editprefset-prefsSafeInfo-widget.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
```

Comment:
Should all source files have a copyright section at the start as what we do for other projects?

Reply:
Definitely. Updated header blocks.